pkg/web_css/lib/src/_search.scss

```diff
@@ -104,3 +104,29 @@
     }
   }
 }
+
+
+.search-completer {
+  overflow-y: auto;
+  overflow-x: hidden;
+  appearance: none;
+  width: 25ex;
+  border: 1px solid #000;
+  max-height: 20em;
+  background-color: #fff;
+  font-size: 16px;
+}
+
+.search-completer-option {
+  cursor: pointer;
+  white-space: nowrap;
+  padding: 0px 3px;
+}
+
+.search-completer-option:hover {
+  background-color: #ccc;
```

Comment:
Note to self: we should have tests that prevent submitting CSS values like this, and forces us to use the color palette that is for both light and dark mode.

This is fine for now, I will update these is a follow-up PR (as I want to also update the naming of the CSS variables).